website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,93 @@
+## Usage
+
+### When to use
+
+- When displaying code examples and longer snippets of code that benefit from syntax highlighting.
+
+### When not to use
+
+- As a full-blown code editor.
+
+### When to use a CopySnippet vs a CodeBlock
+
+There is some overlap in the copying functionality of the CopySnippet and the CodeBlock. Which to use generally comes down to the complexity of the code/value displayed within the component, and whether the user benefits from seeing the larger context of the code example.
+
+For example; if allowing the user to copy an API key or other **singular** value where they don’t necessarily need to see the entire context of where the copied value is expected to be pasted into, use a CopySnippet.
```

Comment:
I _think_ this paragraph needs some refinement, is convoluted and hard to follow. I would use the same format used below: "Use a `CopySnippet` if:"